src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -1152,7 +1152,6 @@ public bool TryReadByteArray(Span<byte> buff, int len)
         // Every time you call this method increment the offset and decrease len by the value of totalRead
         public bool TryReadByteArray(Span<byte> buff, int len, out int totalRead)
         {
-            TdsParser.ReliabilitySection.Assert("unreliable call to ReadByteArray");  // you need to setup for a thread abort somewhere before you call this method
```

Comment:
These shouldn't be removed.

Reply:
yes, there was some mess up,I'am adding them back